test/debugging-test.ts

```diff
@@ -14,12 +14,13 @@ const manifestPath = path.resolve(`${process.cwd()}/test/manifests/test-manifest
 let ws;
 let messageId = 0;
 let connectionOpened = false;
+const limitOfReconnectTries = 25;
 
 async function sleep(ms: number): Promise<any> {
     return new Promise(resolve => setTimeout(resolve, ms));
 }
 
-async function connectToWebsocket(url: string): Promise<any> {
+async function connectToWebsocket(url: string, reconnectTry: number = 1): Promise<any> {
```

Comment:
I was just taking a quick look. You should avoid using `any`. I think here and the sleep function above just needs `Promise<void>` since there is no return value needed.  In TypeScript it is better to use something like "string | boolean" if you need multiple possible types, and in limited cases `unknown` but you should try to avoid that if possible.